pages/linux/pipes.sh.md

```diff
@@ -0,0 +1,26 @@
+# pipess.sh 
+
+> A bash script which draws randomly pathed pipes over the terminal.
+> More information: <https://github.com/pipeseroni/pipes.sh>
+
+- Print the version of the application
+
+`pipes.sh -v`
+
+- Change then Pattern of the pipes (n - 0..8 ):
+
+`pipes.sh -t {{n}}`
```

Comment:
```suggestion
- Change then pa[t]tern of the pipes:

`pipes.sh -t {{0..9}}`
```

[Documentation](https://github.com/pipeseroni/pipes.sh?tab=readme-ov-file#-t--pipe-types) says `0` to `9`